### PR TITLE
feat(admin-ui): add fr translations

### PR DESCRIPTION
**Pull request by gempain**
Add French translations for the Admin UI.

Missing adding the French language to the available languages but needs #5028.

Couldn't find meaningful French equivalents for these:
```
"product-general-section-handle": "Poigné", // Handle
"product-general-section-discountable": "Remisable", // Discountable
"product-raw-section-raw-product": "Produit brut", // Raw product
```

### CODE REVIEW

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 1]
```diff
@@ -0,0 +1,1919 @@
+{
+  "back-button-go-back": "Retourner",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: Precedent?

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 2]
```diff
@@ -0,0 +1,1919 @@
...
+  "analytics-config-form-description": "Vous pouvez choisir d'anonymiser vos données d'utilisation. Si cette option est sélectionnée, nous ne collecterons pas vos informations personnelles, telles que votre nom et votre adresse e-mail.",
+  "analytics-config-form-opt-out": "Désactiver le partage de mes données d'utilisation",
+  "analytics-config-form-opt-out-later": "Vous pouvez toujours désactiver le partage de vos données d'utilisation à tout moment.",
+  "analytics-preferences-success": "Succès",
+  "analytics-preferences-your-preferences-were-successfully-updated": "Vos préférences ont été mises à jour avec succès",
+  "analytics-preferences-error": "Erreur",
+  "analytics-preferences-help-us-get-better": "Aidez-nous à nous améliorer",
+  "analytics-preferences-disclaimer": "Pour créer la meilleure expérience de commerce électronique possible, nous aimerions obtenir des informations sur la manière dont vous utilisez Medusa. Les informations sur les utilisateurs nous permettent de construire des produits meilleurs, plus attrayants et plus utilisables. Nous ne collectons des données que pour améliorer le produit. Lisez quelles données nous recueillons dans notre",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: commerce en ligne?

Comment (adrien2p) [reply packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: Construire de meilleurs produits

Comment (adrien2p) [reply packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: plus engageantes et plus facile d'utilisation?

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 3]
```diff
@@ -0,0 +1,1919 @@
...
+  "product-general-section-metadata": "Métadonnées",
+  "product-general-section-cancel": "Annuler",
+  "product-general-section-save": "Enregistrer",
+  "product-general-section-delete": "Supprimer",
+  "product-general-section-edit-sales-channels": "Modifier les canaux de vente",
+  "product-general-section-published": "Publié",
+  "product-general-section-draft": "Brouillon",
+  "product-general-section-details": "Détails",
+  "product-general-section-subtitle": "Sous-titre",
+  "product-general-section-handle": "Poignée",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: todo: this is a direct translation it does not work. "Identifiant unique" peut etre? @olivermrbl

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 4]
```diff
@@ -0,0 +1,1919 @@
...
+  "product-media-section-edit-media": "Modifier les médias",
+  "product-media-section-upload-images-error": "Une erreur s'est produite lors de la tentative de téléchargement des images.",
+  "product-media-section-file-service-not-configured": "Vous pourriez ne pas avoir de service de fichiers configuré. Veuillez contacter votre administrateur",
+  "product-media-section-error": "Erreur",
+  "product-media-section-media": "Média",
+  "product-media-section-add-images-to-your-product": "Ajoutez des images à votre produit.",
+  "product-media-section-cancel": "Annuler",
+  "product-media-section-save-and-close": "Enregistrer et fermer",
+  "product-raw-section-raw-gift-card": "Carte-cadeau brute",
+  "product-raw-section-raw-product": "Produit brut",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: Donnees Produit?

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 5]
```diff
@@ -0,0 +1,1919 @@
...
+  "product-general-section-sales-channels": "Canaux de vente",
+  "product-media-section-edit-media": "Modifier les médias",
+  "product-media-section-upload-images-error": "Une erreur s'est produite lors de la tentative de téléchargement des images.",
+  "product-media-section-file-service-not-configured": "Vous pourriez ne pas avoir de service de fichiers configuré. Veuillez contacter votre administrateur",
+  "product-media-section-error": "Erreur",
+  "product-media-section-media": "Média",
+  "product-media-section-add-images-to-your-product": "Ajoutez des images à votre produit.",
+  "product-media-section-cancel": "Annuler",
+  "product-media-section-save-and-close": "Enregistrer et fermer",
+  "product-raw-section-raw-gift-card": "Carte-cadeau brute",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: Donnees carte-cadeau?

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 6]
```diff
@@ -0,0 +1,1919 @@
...
+  "product-variants-section-delete-variant-heading": "Supprimer la variante",
+  "product-variants-section-confirm-delete": "Êtes-vous sûr de vouloir supprimer cette variante ?",
+  "product-variants-section-note-deleting-the-variant-will-also-remove-inventory-items-and-levels": " Note : La suppression de la variante supprimera également les éléments et niveaux de l'inventaire",
+  "reset-token-card-error": "Erreur",
+  "reset-token-card-reset-your-password": "Réinitialiser votre mot de passe",
+  "reset-token-card-password-reset-description": "Entrez votre adresse e-mail ci-dessous, et nous vous enverrons des instructions sur la façon de réinitialiser votre mot de passe.",
+  "reset-token-card-email": "E-mail",
+  "reset-token-card-this-is-not-a-valid-email": "Ce n'est pas une adresse e-mail valide",
+  "reset-token-card-send-reset-instructions": "Envoyer les instructions de réinitialisation",
+  "reset-token-card-successfully-sent-you-an-email": "Nous vous avons envoyé un e-mail avec succès",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: suggestion: Un e-mail vous a ete envoye avec succes, je n'ai pas les accents sur mon clavier desole 😂

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 7]
```diff
@@ -0,0 +1,1919 @@
...
+  "templates-province": "Province",
+  "templates-country": "Pays",
+  "templates-metadata": "Métadonnées",
+  "collection-modal-success": "Succès",
+  "collection-modal-successfully-updated-collection": "Collection mise à jour avec succès",
+  "collection-modal-error": "Erreur",
+  "collection-modal-successfully-created-collection": "Collection créée avec succès",
+  "collection-modal-edit-collection": "Modifier la collection",
+  "collection-modal-add-collection": "Ajouter une collection",
+  "collection-modal-description": "Pour créer une collection, vous avez seulement besoin d'un titre et d'une poignée.",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: todo: pareil, poignee est une traduction direct, same as the previous comment about it

File: packages/admin-ui/ui/public/locales/fr/translation.json [hunk 8]
```diff
@@ -0,0 +1,1919 @@
...
+  "collection-modal-details": "Détails",
+  "collection-modal-title-label": "Titre",
+  "collection-modal-title-placeholder": "Lunettes de soleil",
+  "collection-modal-handle-label": "Poignée",
```
Comment (adrien2p) [inline packages/admin-ui/ui/public/locales/fr/translation.json]: todo: same

Comment (adrien2p) [reply packages/admin-ui/ui/public/locales/fr/translation.json]: I let you look at all those poignee

**Review by adrien2p — COMMENTED** [review]
Looks pretty good, I havn't finished the review but here the first few things I ve seen, maybe you can take that as a base to re review the rest?

File: packages/admin-ui/ui/public/locales/fr/translation.json (hunk 7 above)
Comment (gempain) [reply packages/admin-ui/ui/public/locales/fr/translation.json]: cf. PR description. J'ai essayé de trouver mais je ne vois pas ce qu'on appelle "handle" en anglais. Peut-être "ancre" (anchor) ?

**Comment by gempain** [comment]
Rebased on master; added the language to the available languages; Applied changes; Should be good up to line 287.

**Review by olivermrbl — APPROVED** [review]
thanks @gempain - let's get these in and push improvements as they are reported by users 🚀

**Comment by olivermrbl** [comment]
@adrien2p, as discussed, I am merging this now. OK?

**Comment by adrien2p** [comment]
> @adrien2p, as discussed, I am merging this now. OK?

Ok for me 👌

**Pull request merged**